### DISCUSSION TITLE
Access DOM element of Table component

### DISCUSSION THREAD

**Discussion by TryV** [rsuite/rsuite#3038] [Q&A] [answered]
### What problem does this feature solve?
When we try to access the outermost tag, there is no other way but to use the parent element.

### What does the proposed API look like?
It will be useful if we can have ref and className for this component.
For example, to add animation to the table, we need ref.

**Comment by simonguo** [comment, marked as answer]
Table has className and ref props.  However, in order to provide the scrolling method of the table, ref requires .root to get the DOM.

## className 
<img width="971" alt="image" src="https://user-images.githubusercontent.com/1203827/189787596-bee80553-79a2-4663-bd15-2a984da33667.png">


## ref

<img width="940" alt="image" src="https://user-images.githubusercontent.com/1203827/189788096-89740c8d-c80e-4915-9fd4-beebe4140e96.png">

**Comment by SevenOutman** [comment]
I'm converting this issue to discussion as it doesn't seem to be an actual issue
